### ISSUE TITLE
Wrong order of "State 1" and "State 2"  in Figure 29.13 (b) (c) on page 983, Section 29.1.4.5, version 2023-04-01

### ISSUE THREAD

**Issue by zhaoyoubing** [probml/pml2-book#279]
In Figure 29.13 (b) and (c), "state 1" and "state 2"  are actually incorrectly reversed compared to those of (a).

**Comment by murphyk** [comment]
Good catch. I added this clarification to the caption: "Note that in the learned models (b-c), states 1 and 2 are switched compared to the generating model (a),   due to unidentifiability."
